live-build/misc/ansible-roles/appliance-build.zfsonlinux-development/tasks/main.yml

```diff
@@ -65,6 +65,39 @@
     accept_hostkey: yes
     update: no
 
+- shell: ls /lib/modules
+  register: kernel_versions
+
+- shell: |
```

Comment:
I think we also want to add `ignore_errors: yes` (or w/e the syntax is) to prevent zfs build failures from causing appliance-build to fail.

We should probably test and verify that a zfs build failure will be ignored too.